CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [4.4.0] - 2022-06-01
 
+- Added: When using multi-pickup placement, expansions are now considered for logic.
```

Comment:
this could honestly be a major change

Reply:
Remind me of marking relevant changes as major before release.